addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AutomationPanel.java

```diff
@@ -74,6 +78,42 @@
     private static final ImageIcon SAVE_ICON =
             DisplayUtils.getScaledIcon(
                     new ImageIcon(AutomationPanel.class.getResource("/resource/icon/16/096.png")));
+    private static final ImageIcon ADD_PLAN_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "clipboard--plus.png")));
+    private static final ImageIcon DOWN_ARROW_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "task-move-down.png")));
+    private static final ImageIcon UP_ARROW_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "task-move-up.png")));
+    private static final ImageIcon ADD_JOB_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "task--plus.png")));
+    private static final ImageIcon REMOVE_JOB_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "task--minus.png")));
+    private static final ImageIcon ADD_TEST_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "question-plus.png")));
+    private static final ImageIcon REMOVE_TEST_ICON =
+            DisplayUtils.getScaledIcon(
+                    new ImageIcon(
+                            AutomationPanel.class.getResource(
+                                    ExtensionAutomation.RESOURCES_DIR + "question-minus.png")));
```

Comment:
These two files were not added, which causes a NPE and thus the GUI doesnt load at all :(

Reply:
As a temporary work around just make two copies of robot.png and make them as expected. Know that the buttons will have the wrong icon, but the hover text or whatever should be fine. Then you can build the addon yourself and load the locally built one.

Reply:
As a workaround I just commented out the icons and that worked. However, this essentially makes the automation framework GUI changes to be non-existent to end users, right? (Since the panel wont load at all).

Reply:
Or use the task--plus and task--minus files.

Reply:
Good catch!

Reply:
Definitely!